README.md

```diff
@@ -109,10 +106,41 @@ class PostData extends DataTransferObject
 }
 ```
 
-By adding doc blocks to our properties, their values will be validated against the given type; 
-and a `TypeError` will be thrown if the value doesn't comply with the given type.
+Since PHP 7.4 introduced typed properties, you can use the new, built-in syntax. Their values will be validated against the given type and a `TypeError` will be thrown if the value does not comply with it.
 
-Here are the possible ways of declaring types:
+* **NOTE:** Watch out for the upcoming release of PHP 8. More typing features are on the way!
```

Comment:
Are there? 🤔

Reply:
Yup! So far we have union and mixed types coming. This will solve our inability to replicated those doc block types documented.

https://php.watch/versions/8.0/union-types
https://php.watch/versions/8.0/mixed-type